livestream/event_types.go

```diff
@@ -0,0 +1,102 @@
+package livestream
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+)
+
+type (
+	// EventType is the type of a livestream_events record.
+	// This type must stay in sync with the livestream_event_type enum in the database.
```

Comment:
```suggestion
	// EventType is the type of a livestream_events record.
	//
	// This type must stay in sync with the livestream_event_type enum in the database.
```

I can't remember if all IDEs need an empty newline to end the title

Reply:
Seems to be fine in GoLand and in `godoc`:
<img width="745" alt="image" src="https://user-images.githubusercontent.com/2904440/207318716-7da4960d-771d-4bcd-9158-53795c43dc9b.png">
<img width="838" alt="image" src="https://user-images.githubusercontent.com/2904440/207319227-1559f162-5471-47e4-9339-e117161a7701.png">